apiary.apib

```diff
@@ -3141,6 +3142,52 @@ Update a draft invoice. Booked invoices cannot be updated.
 
 + Response 204
 
+### invoices.send [POST /invoices.send]
+
+Send an invoice via e-mail.
+
++ Request (application/json)
+    + Attributes (object)
+        + id: `d885e5d5-bacb-4607-bde9-abc4a04a901b` (string, required)
+        + content (object, required)
+             + subject: `Invoice` (string, required)
+             + body: `Please find your invoice attached to this mail` (string, required)
+             + template_id: `045cd6a9-7527-09c9-aa16-fd5ac7953e71` (string, optional, nullable)
```

Comment:
Q: What is the difference between quotations & invoices?
Aren't quotations also send with templates?

Can you explain why is there a template id option here and not on quotations?

Reply:
This structure is also not matching the structure we documented on`/quotations.send`, there it isn't wrapped in `content` at all.

Let's make this consistent by updating the structure here or on the just published `/quotations.send`